code/game/gamemodes/game_mode.dm

```diff
@@ -241,8 +241,8 @@
 /*	for(var/mob/M in player_list)
 		if(!istype(M,/mob/new_player))
 			M << sound('sound/AI/intercept.ogg')
-	if(security_level < SEC_LEVEL_BLUE)
-		set_security_level(SEC_LEVEL_BLUE)*/
+	if(security_level < SEC_LEVEL_SUNSET)
```

Comment:
This is just commented out code. Better to just remove this - it's probably ancient.